pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -402,6 +403,33 @@ func (a v1alpha1PolicyApplier) InboundFilterChain(sdsUdsPath string, meta *model
 				FilterChainMatch: &ldsv2.FilterChainMatch{},
 			},
 		}
+		if meta.DNSCert != "" {
```

Comment:
Does this mean, MTLS with DNSCert is only available for `PERMISSIVE` mode?

Can I client not pass non SPIFFE format certs for a STRICT client?